ParcelInvoiceMaker/ParcelInvoiceMaker/Model/Discount.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Discount.swift
+//  ParcelInvoiceMaker
+//
+//  Created by Hong yujin on 4/12/24.
+//
+
+import Foundation
+
+enum Discount: Int {
+    case none, vip, coupon
+    var strategy: DiscountStrategy {
+        switch self {
+        case .none:
+            NoneStrategy()
+        case .vip:
+            VipStrategy()
+        case .coupon:
+            CouponStrategy()
+        }
+    }
+}
+
+protocol DiscountStrategy {
+    func applyDiscount(deliveryCost: Int) -> Int
+}
+
+private struct NoneStrategy: DiscountStrategy { 
```

Comment:
NoneStrategy라는 객체가 무슨 역할인지 조금 헷갈리는거 같아 제안드려봅니다~

```suggestion
private struct NoDiscountStrategy: DiscountStrategy { 
```